client/tests/integration/unstable_network.rs

```diff
@@ -16,63 +18,33 @@ const MAXIMUM_TRANSACTIONS_IN_BLOCK: u32 = 5;
 fn unstable_network_4_peers_1_fault() {
     let n_peers = 4;
     let n_transactions = 20;
-    // Given that the topology will resolve view changes by shift for `n_peers` consequent times
-    // and having only 1 faulty peer guarantees us that at maximum in `n_peers` tries the block will be committed.
-    // So for the worst case scenario for `n_transactions` given 1 transaction per block
-    // the network will commit all of them in `n_peers * n_transactions` tries.
```

Comment:
Correct me if I'm wrong, but it feels that the condition described here hasn't changed, so I don't see a reason to remove the comment. Additional clarifications never hurt.

Reply:
Not in this PR. But sumeragi changes have already been merged that make this comment out of date.